charts/hpe-csi-driver/hpe-csi-driver/2.0.0/values.schema.json

```
@@ -0,0 +1,169 @@
+            "type": "string",
+            "default": "IfNotPresent",
+            "enum": [ "Always", "IfNotPresent", "Never" ]
+        },
+        "iscsi": {
+            "$id": "#/properties/iscsi",
+            "title": "iSCSI CHAP credentials",
+            "type": "object",
+            "default": 
+                {
+                    "chapUser": "",
+                    "chapPassword": ""
+                },
+            "required": [
+                "chapUser",
+                "chapPassword"
+            ],
+            "properties": {
+                "chapUser": {
+                    "$id": "#/properties/iscsi/properties/chapUser",
+                    "title": "CHAP username",
+                    "type": "string",
+                    "default": ""
+                },
+                "chapPassword": {
+                    "$id": "#/properties/iscsi/properties/chapPassword",
+                    "title": "CHAP password",
+                    "description": "Between 12 and 16 characters",
+                    "type": "string",
+                    "default": "",
+                    "pattern": "^$|^[a-zA-Z0-9+_)(*^%$#@!]{12,16}$"
+                }
+            },
+            "additionalProperties": false
+        },
+        "logLevel": {
+            "$id": "#/properties/logLevel",
+            "title": "Set the log level of the HPE CSI Driver images",
+            "type": "string",
+            "default": "info",
+            "enum": [ "info", "debug", "trace", "warn", "error" ]
+        },
+        "registry": {
+            "$id": "#/properties/registry",
+            "title": "Pull images from a different registry than default",
+            "description": "SIG Storage images needs to be mirrored from k8s.gcr.io to this registry if this parameter is changed.",
+            "type": "string",
+            "default": "quay.io"
+        },
+        "kubeletRootDir": {
+            "$id": "#/properties/kubeletRootDir",
+            "title": "Kubelet root directory",
+            "description": "Only change this if the kubelet root dir has been altered by the Kubernetes platform installer.",
+            "type": "string",
+            "default": "/var/lib/kubelet",
+            "pattern": "^/"
+        },
+        "disableNodeGetVolumeStats": {
+            "$id": "#/properties/disableNodeGetVolumeStats",
+            "title": "Disable the CSI nodeGetVolumeStats call",
+            "description": "In very large environments, disabling this feature may alleviate pressure on the CSP.",
+            "type": "boolean",
+            "default": false
+        },
+        "cspClientTimeout": {
+            "$id": "#/properties/cspClientTimeout",
+            "title": "CSP client timeout",
+            "description": "Change the CSP client timeout for HPE Alletra 9000, Primera and 3PAR systems",
+            "type": "integer",
+            "default": 60,
+            "minimum": 60,
+            "maximum": 360
+        }
```

Please add the following to take care of the issue described in the other comment. Do not modify this file directly, instead make this change in the generated charts directory (after running `make prepare`) in order to save it in the patch.
```suggestion
        },
        "global": {}